R/test-plotDecision.R

```diff
@@ -0,0 +1,23 @@
+# plotDecision ----
+testthat::test_that("plotDecision gives a correct result", {
```

Comment:
```suggestion
test_that("plotDecision gives a correct result", {
```
etc below, or did we always use that prefix?

Reply:
no we didn't I was trying something as there was an error that I couldn't explain at one point

Reply:
fixing this at present, it won't work WITHOUT the `testthat::` weirdly